lib/webauthn/configuration.rb

```diff
@@ -27,19 +29,28 @@ def self.if_pss_supported(algorithm)
     attr_accessor :verify_attestation_statement
     attr_accessor :credential_options_timeout
     attr_accessor :silent_authentication
+    attr_accessor :acceptable_attestation_types
+    attr_reader :attestation_root_certificates_store
 
     def initialize
       @algorithms = DEFAULT_ALGORITHMS.dup
       @encoding = WebAuthn::Encoder::STANDARD_ENCODING
       @verify_attestation_statement = true
       @credential_options_timeout = 120000
       @silent_authentication = false
+      @acceptable_attestation_types = [:None, :Self, :Basic]
```

Comment:
Seems like we're missing [`:AttCA`](https://www.w3.org/TR/webauthn/#sctn-attestation-types)? Same in `valid_attestation_trustworthiness?`

ECDAA is safe to ignore for now as no-one has implemented it.

Reply:
You're right! Same for `:Basic_or_AttCA`